supervised-learning/boosted_trees_regression.md

```diff
@@ -1,4 +1,5 @@
-# Random Forest Regression Trees
+# Gradient Boosted Regression Trees
+
 The Gradient Boosted Regression Trees (GBRT) model (also called Gradient
 Boosted Machine or GBM), along with Random Forest, is one of the most effective machine learning
```

Comment:
"random forest" isn't capitalized in the chapter on random forest regression - not sure it should be here either.